### PR TITLE
add session configs to example bots

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- avoid these annoying warnings 
```
/Users/ella/RasaHQ/rasa/rasa/core/domain.py:151: FutureWarning: No tracker session configuration was found in the loaded domain. Domains without a session config will automatically receive a session expiration time of 60 minutes in Rasa version 2.0 if not configured otherwise.
  session_config = cls._get_session_config(data.get(SESSION_CONFIG_KEY, {}))
2020-03-25 20:29:52 INFO     rasa.model  - Data (version) for Core model section changed.
2020-03-25 20:29:52 INFO     rasa.model  - Data (version) for NLU model section changed.
Training Core model...
/Users/ella/RasaHQ/rasa/rasa/core/domain.py:151: FutureWarning: No tracker session configuration was found in the loaded domain. Domains without a session config will automatically receive a session expiration time of 60 minutes in Rasa version 2.0 if not configured otherwise.
  session_config = cls._get_session_config(data.get(SESSION_CONFIG_KEY, {}))
```

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: examples/reminderbot/domain.yml [hunk 1]
```diff
@@ -17,19 +14,25 @@ intents:
...
   utter_goodbye:
   - text: Bye
+
 actions:
 - action_set_reminder
 - action_forget_reminders
 - action_react_to_reminder
 - action_tell_id
 - action_warn_dry
-- utter_what_can_do
```
Comment (melindaloubser1) [inline examples/reminderbot/domain.yml]: Why were these deleted?

Comment (erohmensing) [reply examples/reminderbot/domain.yml]: This new example bot wasn't merged yet when these changes were made https://github.com/RasaHQ/rasa/pull/5083/files
